packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -42,6 +43,10 @@ export interface OperatorPluginConfig {
     inspectRandomNode: {
         intervalInMs: number
     }
+    closeExpiredFlags: {
+        intervalInMs: number
+        maxFlagAgeInSec: number
```

Comment:
The `flag` word is redundant, and I'd consider changing the data unit to milliseconds. For users perspective it is easier if all our config parameters have the same unit. (of course the field name tells the unit, but anyway easier to have just milliseconds everywhere)

```suggestion
        maxAgeInMs: number
```